### ISSUE TITLE
Suggestion: Brick Version Requirements.

### ISSUE THREAD

**Issue by rnlambert7** [Gottwik/Enduro#218]
TL:DR; Bricks need a minimum/maximum version requirement for Enduro.

After the recent update in 1.4.45 Bricks can use Admin Settings from `enduro.json`: `https://github.com/Gottwik/Enduro/commit/91e9c7869dd251d7c3fc047548f06cc6ff3b943e`

However I can still install a brick and there is no notice that any part of the code would break because of a version change. 

To be more specific and give an example, for `enduro_quill` the variable `brick_admin_settings` located at `https://github.com/Gottwik/enduro_quill/blob/master/assets/inject_quill_to_admin.js#L14` now causes a JS error in the console on admin edit screen where quill is being used.

A version requirement for a brick's usage would allow a user to know whether or not they could utilize a brick, and then disable it should there be a mismatch.
